nservicebus/messaging/data-distribution.md

```diff
@@ -1,40 +1,40 @@
 ---
 title: Data distribution
 summary: Why asynchronous messaging is not a good fit for data distribution scenarios
-reviewed: 2020-05-21
+reviewed: 2020-09-01
 redirects:
 - nservicebus/azure/data-distribution
 - samples/routing/data-distribution
 ---
 
 Data distribution refers to a pattern where asynchronous messages (e.g. using NServiceBus) are used to deliver data to multiple targets in some coordinated fashion.
 
-A common example of a data distribution scenario is having cached data on multiple scaled-out web servers, and then attempting to use messages delivered to all web servers in a cache-busting fashion, so that each web server drops its current cache entries, and then is forced to retrieve fresh data from the database.
+A common example of a data distribution scenario is having cached data on multiple, scaled-out web servers, and attempting to deliver a message to each of them which indicates that they should drop their current cache entries and retrieve fresh data from the database.
```

Comment:
I think comma placement here is an example of the putting-a-comma-where-you-would-pause-when-reading sin, not to actually separate fragments. It's long anyway - how about 2 sentences.
```suggestion
A common example of a data distribution scenario is having cached data on multiple scaled-out web servers and attempting to deliver a message to each of them. The message indicates each server should drop their current cache entries and retrieve fresh data from the database.
```

Reply:
> I think comma placement here is an example of the putting-a-comma-where-you-would-pause-when-reading sin, not to actually separate fragments.

I thought that the two adjectives should have commas in-between, but it turns out that's only the case when they are of [equal rank](https://www.grammarflip.com/curriculum/commas-between-two-adjectives), which these are not.

> It's long anyway - how about 2 sentences.

👍